src/main/java/seedu/address/ui/TaskCard.java

```diff
@@ -50,7 +50,7 @@ public TaskCard(Task task, int displayedIndex) {
         address.setText(task.getDescription().value);
         email.setText(task.getPriorityValue().value);
         status.setText(task.getStatus().toString());
-        task.getLabels().forEach(tag -> tags.getChildren().add(new Label(tag.labelName)));
+        task.getLabels().forEach(label -> tags.getChildren().add(new Label(label.labelName)));
```

Comment:
There's still a "tags" in here?

Reply:
The rest of the file is untouched in terms of refactoring. That being said, you make a good point -- I don't think the local name here should be changed, everything should be kept as it is, leaving a new PR to fix all the properties used in this file